draft-specifications/Model.md

```diff
@@ -407,6 +486,22 @@ A local target handles deliveries and produces either a return value
 (fulfillment) or thrown error (rejection reason) for a message delivery.
 A remote target (a presence) forwards messages to its corresponding local target.
 
+The Presentation Format for a target is a [Record](Formats.md#record) with two
+values.
+The first value is a [Selector](Formats.md#selector) indicating whether the
+value corresponds to a local or remote target.
+The second value is a positive [Integer](Formats.md#integer) specifying a
+target as tracked by the sender or receiver.
+
+If the Selector is spelled `desc:import-object`, the target is local to the
+sender and remote to the receiver of the value.
+If the Selector is spelled `desc:export`, the target is remote to the
+sender and local to the receiver of the value.
```

Comment:
What about the three party case, where the reference to Carol in VatC is remote to both Alice in VatA and Bob in VatB. What ocapn does vatA send to vatB to talk about the mutually remote target?

Reply:
My understanding is that the transfer occurs out-of-band, but I have not attempted to implement 3PH based on the CapTP spec myself and so defer to @tsyesika. Does the in-band data model require special records for 3PH or will these suffice? https://github.com/ocapn/ocapn/blob/main/draft-specifications/CapTP%20Specification.md#third-party-handoffs